[2021-08-12 13:09:50,170] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T13:09:43.681984+00:00 [queued]>
[2021-08-12 13:09:50,207] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T13:09:43.681984+00:00 [queued]>
[2021-08-12 13:09:50,208] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 13:09:50,208] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-12 13:09:50,209] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 13:09:50,230] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): thursday_Task> on 2021-08-12T13:09:43.681984+00:00
[2021-08-12 13:09:50,245] {standard_task_runner.py:52} INFO - Started process 30191 to run task
[2021-08-12 13:09:50,254] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pipeline_try', 'thursday_Task', '2021-08-12T13:09:43.681984+00:00', '--job-id', '88', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_try1.py', '--cfg-path', '/tmp/tmpc2me5pv8', '--error-file', '/tmp/tmp6p689mtw']
[2021-08-12 13:09:50,259] {standard_task_runner.py:77} INFO - Job 88: Subtask thursday_Task
[2021-08-12 13:09:50,311] {logging_mixin.py:104} INFO - Running <TaskInstance: pipeline_try.thursday_Task 2021-08-12T13:09:43.681984+00:00 [running]> on host 99300daf4d26
[2021-08-12 13:09:50,374] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=ameysonje242000@gmail.com
AIRFLOW_CTX_DAG_OWNER=amey
AIRFLOW_CTX_DAG_ID=pipeline_try
AIRFLOW_CTX_TASK_ID=thursday_Task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T13:09:43.681984+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T13:09:43.681984+00:00
[2021-08-12 13:09:50,376] {logging_mixin.py:104} INFO - From thursday_Task, so its thursday today
[2021-08-12 13:09:50,377] {python.py:151} INFO - Done. Returned value was: httpposttry
[2021-08-12 13:09:50,412] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=pipeline_try, task_id=thursday_Task, execution_date=20210812T130943, start_date=20210812T130950, end_date=20210812T130950
[2021-08-12 13:09:50,459] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-12 13:09:50,505] {local_task_job.py:149} INFO - Task exited with return code 0
